lib/SILOptimizer/Differentiation/PullbackCloner.cpp

```diff
@@ -1722,21 +1722,28 @@ bool PullbackCloner::Implementation::run() {
   }
 
   // Create pullback blocks and arguments, visiting original blocks in
-  // post-order post-dominance order.
-  SmallVector<SILBasicBlock *, 8> postOrderPostDomOrder;
-  // Start from the root node, which may have a marker `nullptr` block if
-  // there are multiple roots.
-  PostOrderPostDominanceOrder postDomOrder(postDomInfo->getRootNode(),
-                                           postOrderInfo, original.size());
```

Comment:
Can you please delete `PostOrderPostDominanceOrder` (a differentiation-specific utility), now that it's no longer used?